tailwind.config.js

```diff
@@ -1,33 +1,23 @@
-const plugin = require('tailwindcss/plugin')
+const plugin = require("tailwindcss/plugin");
 
 module.exports = {
-    mode: 'jit',
-    purge: [
-        './**/*.html',
-        './**/*.ts',
-    ],
-    darkMode: false, // or 'media' or 'class'
-    theme: {
-        extend: {
-            maxHeight: {
-                '65vh': '65vh',
-                '20vh': '20vh',
-            }
-        },
+  mode: "jit",
+  content: ["./**/*.html", "./**/*.ts"],
+  theme: {
+    extend: {
+      maxHeight: {
+        "65vh": "65vh",
+        "20vh": "20vh",
+      },
     },
-    variants: {
-        extend: {
-            ringColor: ['hover'],
-        }
-    },
-    plugins: [
-        plugin(function ({addVariant, e}) {
-            addVariant('landscape', ({modifySelectors, separator}) => {
-                modifySelectors(({className}) => {
-                    return `.${e(`landscape${separator}${className}`)}:landscape`
-                })
-            })
-        })
-
-    ]
-}
+  },
+  plugins: [
+    plugin(function ({ addVariant, e }) {
+      addVariant("landscape", ({ modifySelectors, separator }) => {
+        modifySelectors(({ className }) => {
+          return `.${e(`landscape${separator}${className}`)}:landscape`;
+        });
+      });
+    }),
+  ],
```

Comment:
This is something to double check at some point. I cannot remember why we needed this as a plugin vs. a regular breakpoint definition.

It would be forth checking again if this still works. When I added TW I only found out about this very late because it is well hidden. UseCase AFAIR is iPhone in landscape view.